packages/excalidraw/components/App.tsx

```diff
@@ -5195,7 +5198,12 @@ class App extends React.Component<AppProps, AppState> {
       event.clientY - this.state.offsetTop,
     );
     const isOverScrollBar = isPointerOverScrollBars.isOverEither;
-    if (!this.state.newElement && !this.state.multiElement) {
+    if (
+      !this.state.newElement &&
+      !this.state.selectionElement &&
+      !this.state.selectedElementsAreBeingDragged &&
+      !this.state.multiElement
+    ) {
```

Comment:
this doesn't have any effect right now, but if we later have a `:hover` effect when over scrollbars or something, it would matter. So adding the checks to keep parity with prod.

Reply:
let's not write test for this. It's actaully just half the story, and scrollbars are broken anywa.